tests/tests/checks/snapshots/analysis/js/import.mjs/check_glob@imported_symbols__json.snap

```diff
@@ -0,0 +1,118 @@
+---
+source: tests/test_check_insta.rs
+expression: "&json_results"
+---
+[
+  {
+    "loc": "00001:0",
+    "target": 1,
+    "kind": "def",
+    "pretty": "file js/import.mjs",
+    "sym": "FILE_js/import@2Emjs"
+  },
+  {
+    "loc": "1:7-9",
+    "source": 1,
+    "syntax": "def,prop",
+    "pretty": "property ns",
+    "sym": "#ns"
+  },
+  {
+    "loc": "1:7-9",
+    "target": 1,
+    "kind": "def",
+    "pretty": "ns",
+    "sym": "#ns",
+    "context": ""
+  },
```

Comment:
So we could use a "decl" kind here, but again, probably having import/export might work better.  I guess a fun thing that could be done here is also that if we baked the file path into the symbols we could also have the jumpref mechanism (which I think I discussed somewhat recently with Botond in scrollback in the channel) then provide for "go to export".  That is, right now `#ns` is of course not unique, but `FILEPATH#ns` is and there should be a unique pair between the import and the export making it easy for jumpref and the context-menu to do the right thing.

It's also fine if we don't do that, as this potentially gets into overlap with scip-typescript, but we would likely end up modeling it in a similar fashion in that case.  (The main difference would just be the symbols that we assign in that case.)

I realize you may already have plans for something like this, in which case, consider this a confirmation of the wiseness of your plans :)

Reply:
Yeah, adding "go to export" would be really helpful.
I'll look into it after [bug 1895653](https://bugzilla.mozilla.org/show_bug.cgi?id=1895653) to also handle relative paths cases.